### ISSUE TITLE
Limit Exceeded: URLFetch URL Length. in updateDocument with mask=true

### ISSUE THREAD

**Issue by youleaf** [grahamearley/FirestoreGoogleAppsScript#54]
When calling updateDocument with mask=true and a big set of fields to update, i got 

> Error in firestore.updateDocument: {"message":"Limit Exceeded: URLFetch URL Length.","name":"Exception","fileName":"Util (FirestoreApp)","lineNumber":25,"stack":"\tat Util (FirestoreApp):25 (fetchObject_)\n\tat Request (FirestoreApp):46\n\tat Request (FirestoreApp):98\n\tat Write (FirestoreApp):48 (updateDocument_)\n\tat Firestore (FirestoreApp):87\n}

There is an hard limit in URLFetch about url lenght:
https://issuetracker.google.com/issues/35892557
(This feature request is pending since 2012)

Inspecting the problem I noticed that fields to be masked in the update are added as request params in Write.js:

https://github.com/grahamearley/FirestoreGoogleAppsScript/blob/97b3b3c5468a9fa4d0e0b9906e896a4e86522db0/Write.js#L42

Most probably my request exceeds the hard limit (in my use case, I need to update 60 over 63 fields in my doc, so the resulting url contains tens of fields).

I'm using version 22.

**Comment by LaughDonor** [comment]
This isn't a limitation of the library itself. However, since you are only updating certain fields, perhaps the best workaround (if anyone else comes across this) would be to split up the amount of fields to be updated and call `.updateDocument` multiple times.

**Comment by j05u3** [comment]
May I suggest that another way around it would be [this](https://stackoverflow.com/a/59681235/2469583)

**Comment by BoomerangTransport** [comment]
Hi everyone.  I ran into the exact same issue, so I'm glad that this thread is here.  Thank you lelesrc for raising it and explaining it so well.  I appreciate LaughDonor acknowledging the issue as well and providing a possible workaround.  Thank you to j05u3 for providing another possible workaround, which seems to be meant more for the library itself.  

For anyone that runs into the same issue- I first solved this by creating a custom set of Firebase Functions that accomplished the same goal.  That worked.  But after creating them, I came back to this, found this thread, and set the mask parameter to false.  Handpalm, that worked too.  My guess is that the mask adds the fields to the request url from the body OR adds additional characters to the request that in my case pushed it just past the limit.  Either way, in my one example, that fix happened to work.  I'm leaving this in case it helps anyone else or if anyone has more detailed thoughts on how to solve / workaround this issue.
